### ISSUE TITLE
SR doesn't read when page loads more data from api

### ISSUE THREAD

**Issue by alekyah-springml** [nvaccess/nvda#13015]
### Steps to reproduce:
Go to https://getvaccinated.oregon.gov/#/
Click 'View Vaccine Locator Map' button.
Wait until the pins load on the map.

### Actual behavior:
After pins have loaded on the map, click tab to go through the items. NVDA screen reader doesn't read any of the items (links, buttons, etc). 

When there are less than 400 events on the map, the Screen Reader is perfectly reading the text when tabbing through the items. But when the events are more than 400, the SR doesn't read the text

### Expected behavior:
When pins have loaded on the map, tabbing through the items should read out the text even if there are more events.

#### NVDA installed/portable/running from source:
Edition	Windows 10 Home Single Language
Version	20H2
OS build	19042.1288

#### Windows version:
Edition	Windows 10 Home Single Language
Version	20H2
OS build	19042.1288

#### Does the issue still occur after restarting your computer?
YES

**Comment by ehollig** [comment]
@alekyah-springml it appears this page may have been updated. Are you still experiencing this issue? Also, which browser and version number of the browser and NVDA are you running?

**Comment by seanbudd** [comment]
Welcome @alekyah-springml 

This is very likely a problem with the implementation of the application/website, as a starting point please submit feedback to the authors to let them know about the problem.
Considering the age of the issue, and that it may have already been fixed by the author, so we are closing this issue.
Please let us know if it is still an issue, and we can re-open it.

Kind Regards,
NV Access Software Developers
